### DISCUSSION TITLE
No connection to public server possible under any conditions (since version 1.2.3-1)

### DISCUSSION THREAD

**Discussion by LinguaOccultus** [rustdesk/rustdesk#8736] [General]
### Bug Description

Hello,

using RustDesk for some time now and began with version 1.2.3 when everything was fine. Since version 1.2.3-1, however, I cannot connect to the public server anymore whatever I do. RustDesk is the only application having network/internet issues while everything else still works fine, so I guess it is an issue directly related to it and/or the public server.

I was not aware of this issue first, since I mostly use it at the moment to remote control devices by direct IP connection inside my private/personal network. But I do have an RSS feed set on your GitHub release to be notified of updates and when I tried to update RustDesk, I had to do so manually at the end by uninstall/reinstall due to the connection issues. Just in case that helps somehow with understanding ... ;-)

I did some test which results are as follows (I highlighted and commented to make things a bit more clear):

**C:\Users\Tim>nslookup [rs-ny.rustdesk.com](http://rs-ny.rustdesk.com/)**
Server: [dns9.quad9.net](http://dns9.quad9.net/)
Address: 9.9.9.9

Nicht autorisierende Antwort:
Name: [rs-ny.rustdesk.com](http://rs-ny.rustdesk.com/)
Address: 209.250.254.15

**C:\Users\Tim>ping [rs-ny.rustdesk.com](http://rs-ny.rustdesk.com/)**

Ping wird ausgeführt für [rs-ny.rustdesk.com](http://rs-ny.rustdesk.com/) [209.250.254.15] mit 32 Bytes Daten:
Zeitüberschreitung der Anforderung. **(No ping possible already)**
Zeitüberschreitung der Anforderung.
Zeitüberschreitung der Anforderung.
Zeitüberschreitung der Anforderung.

Ping-Statistik für [209.250.254.15](http://209.250.254.15/):
Pakete: Gesendet = 4, Empfangen = 0, Verloren = 4
(100% Verlust),

**C:\Users\Tim>tracert [rs-ny.rustdesk.com](http://rs-ny.rustdesk.com/)**

Routenverfolgung zu [rs-ny.rustdesk.com](http://rs-ny.rustdesk.com/) [209.250.254.15]
über maximal 30 Hops:

1 <1 ms <1 ms <1 ms SYNOLOGYROUTER [192.168.1.1] **(Endpoint of my network)**
2 1 ms 2 ms 2 ms 192.168.0.1 **(My ISP)**
3 16 ms 14 ms 11 ms [ip-081-210-176-170.um21.pools.vodafone-ip.de](http://ip-081-210-176-170.um21.pools.vodafone-ip.de/) [81.210.176.170] **(Public internet from here)**
4 18 ms 26 ms 16 ms [de-fra01b-rc2-ae-24-0.aorta.net](http://de-fra01b-rc2-ae-24-0.aorta.net/) [84.116.196.190]
5 15 ms 21 ms 18 ms [de-bfe18a-rt01-lag-1.aorta.net](http://de-bfe18a-rt01-lag-1.aorta.net/) [84.116.190.34]
6 15 ms 25 ms 17 ms [ae8-100-tcr1.fnt.cw.net](http://ae8-100-tcr1.fnt.cw.net/) [195.2.26.93]
7 19 ms 22 ms 19 ms [gtt-gw-tcr1.fnt.cw.net](http://gtt-gw-tcr1.fnt.cw.net/) [195.2.6.66]
8 30 ms 29 ms 31 ms [ae2.cr2-ams13.ip4.gtt.net](http://ae2.cr2-ams13.ip4.gtt.net/) [141.136.106.174]
9 * * * Zeitüberschreitung der Anforderung. **(From this point, something stopped working)**
10 * * * Zeitüberschreitung der Anforderung.
11 * * * Zeitüberschreitung der Anforderung ...

As I have read some people are discussing DNS is causing this, I did contact the Support of Quad9 which free public 9.9.9.9 DNS is the one I am using. They received the informations above also, of course. However, their response was as follows:

_»Hello,
 
Thanks for contacting Quad9 support.
 
ICMP doesn't necessarily indicate a problem, as some networks and hosts block ICMP traffic.
 
Quad9 in this case is doing its job. We're returning the IP addresses of the domain name that's being queried.
 
The traceroute indicates that the IP address may be unrouteable when routing via GTT (hop [#8](https://quad9.zendesk.com/tickets/8)).
 
The IP address of the A record appears to be correct. If this is not the expected IP address, then Rustdesk would need to open a support ticket with their authoritative DNS provider.
 
$ dig +short [rs-ny.rustdesk.com](http://rs-ny.rustdesk.com/) @[1.1.1.1](http://1.1.1.1/)
209.250.254.15

$ dig +short [rs-ny.rustdesk.com](http://rs-ny.rustdesk.com/) @[9.9.9.9](http://9.9.9.9/)
209.250.254.15

$ dig +short [rs-ny.rustdesk.com](http://rs-ny.rustdesk.com/) @[8.8.8.8](http://8.8.8.8/)
209.250.254.15
 
In this case, Quad9 is behaving as expected.«_

Any help? I do have quite a bit of network knowledge and experience and checked my router and connection and everything, but all is working fine and as aspected as every other application and device on my network and/or internet connection is, except RustDesk since version 1.2.3-1 (or whatever changed on your side regarding the public servers).

Best regards,

**_Tim_**

### How to Reproduce

Not possible as I am just starting RustDesk and immediately receive this status at the bottom of its window. I guess, this can only be reproduced on my site if there is no general problem (see above).

If you need me to test something, just tell me ... !

### Expected Behavior

Obvious.

### Operating system(s) on local side and remote side

Windows 11 Pro 23H2

### RustDesk Version(s) on local side and remote side

1.2.6

### Screenshots

None needed. Can do one from the main window but it just shows that it has not server connection and it looks the very same as in all the other reports of this issue in the discussions here ...

### Additional Context

None at the moment (see above).

If needed, just ask me ... !

**Comment by DevBefell** [comment]
I have a the same problem as well,  using Quad9's DNS on Windows 11 denies the relay server for RustDesk.
This might be a Quad9 issue but best to investigate further, as for now use a different DNS provider.

**Comment by LinguaOccultus** [comment]
Just tested with 1.2.7 which now can be simpler installed on top of the existing installation, finally. However, still the same issue ...

As you can read in my initial post, I did contact Quad9 and based on my information above, they said the issue is not on their side. But when I have a bit spare time, I will do test other DNS and report the results here ...

Also, this WORKED with the prior version on Quad9 before, as mention also above, but sure, there could be changes meanwhile on their side also ...

**Comment by LinguaOccultus** [comment]
Just found out that connections to Android (x64, also 1.2.7) are not possible anymore also. No detailed error message. For the moment, only IP connections inside my own network are possible. All that worked with version 1.2.3 before and nothing has changed since then on my side (aside the RustDesk version updates, of course).

**Comment by LinguaOccultus** [comment]
Out of nowhere and without ANY changes on my end, the server connection is working again so I guess, there was a known issue on the server side which is solved. Sad, that there is so bad communication about this. =(

Closed due to solved for me (for now, at least).
